kolibri/plugins/facility_management/assets/src/views/user-page/edit-user-modal.vue

```diff
@@ -184,11 +185,22 @@
       nameIsInvalid() {
         return Boolean(this.nameIsInvalidText);
       },
+      usernameAlreadyExists() {
+        return this.facilityUsers.find(
+          ({ username }) => username.toLowerCase() === this.newUsername.toLowerCase()
+        );
+      },
       usernameIsInvalidText() {
         if (this.usernameBlurred || this.formSubmitted) {
+          if (this.username.toLowerCase() === this.newUsername.toLowerCase()) {
```

Comment:
Just checks whether the username was never changed or just changed its capitalization.

if the current username is `e` and changed to `E`, `Username already exists` error will display

Reply:
I see. The way you explain it, it might make more sense to put this logic in `userNameAlreadyExists`, right before you search through `facilityUsers`

If old user name is `e` and the new one is `E`, then return from `userNameAlreadyExists` with `false`.

Reply:
Oh! I guess you're right. I just added it there for a little speed

I'll make the changes